### DISCUSSION TITLE
French translation

### DISCUSSION THREAD

**Discussion by obones** [toverainc/willow#146] [General]
I see on the readme that you are looking for translation volunteers, so once you have selected a Unicode font, do not hesitate to ask me, I've been doing French translations for projects I take part in for the past 20 years.

**Comment by kristiankielhofner** [comment]
We have the unicode font working in latest main. Our current main task is implementing dynamic configuration and flashing of pre-built images so users don't need any of the build steps, configuration, etc.

Once this is completed implementing support for multiple UI languages becomes much easier, and we would definitely appreciate your help with French!
